cdc/puller/mock_puller.go

```diff
@@ -13,321 +13,321 @@
 
 package puller
 
-import (
```

Comment:
i comment this because there is no `github.com/pingcap/tidb/util/testkit` in tidb now,
what's more, there is no reference to this package or struct in our repo, 

all usage before is in `mock_puller_test.go` but this test file is also commented in https://github.com/pingcap/tiflow/pull/313